### ISSUE TITLE
Schema aggregator: $table->ulid() default column name maps to uuid instead of ulid

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#530]
## Description

`METHODS_HAVE_DEFAULT_COLUMN_NAME` maps `'ulid' => 'uuid'` (line 43 of SchemaAggregator.php), but Laravel's `Blueprint::ulid()` defaults to column name `'ulid'`, not `'uuid'`.

## Laravel source (present in both Laravel 12 and 13)

```php
// Blueprint.php
public function ulid($column = 'ulid', $length = 26)
{
    return $this->char($column, $length);
}
```

The default column name is `'ulid'`, but the plugin registers it as `'uuid'`.

## Impact

`$table->ulid()` (without arguments) registers a column named `uuid` instead of `ulid`. Models with a `ulid` column will have the wrong column name in the schema, potentially causing false positives or false negatives.

## Suggested fix

```diff
- 'ulid' => 'uuid',
+ 'ulid' => 'ulid',
```
